packages/web-components/src/components/ic-top-navigation/ic-top-navigation.stories.mdx

```diff
@@ -1,36 +1,21 @@
-import {
-  Meta,
-  Story,
-  Canvas,
-  ArgsTable,
-  Description,
-} from "@storybook/addon-docs";
-import { html } from "lit-html";
-import readme from "./readme.md";
-import NavigationButton from "../ic-navigation-button/readme.md";
-import NavigationItem from "../ic-navigation-item/readme.md";
-import NavigationGroup from "../ic-navigation-group/readme.md";
+import { Meta, Story, Canvas, ArgsTable, Description } from '@storybook/addon-docs';
```

Comment:
Looks like some of the formatting in this file has changed - running 'npm run prettier:fix' should fix it 🙂